docs/source/deployment/databricks/databricks_deployment_workflow.md

```diff
@@ -95,6 +95,10 @@ kedro new --starter=databricks-iris
 
 This command creates a new Kedro project using the `databricks-iris` starter template. Name your new project `iris-databricks` for consistency with the rest of this guide.
 
+ ```{note}
+ If you are not using the `databricks-iris` starter to create a Kedro project, then you should [disable file-based logging](../../logging/logging.md#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
```

Comment:
```suggestion
 If you are not using the `databricks-iris` starter to create a Kedro project, then you should [disable file-based logging](../../logging/index.md#disable-file-based-logging) to prevent Kedro from attempting to write to the read-only file system.
```